manticore/platforms/linux.py

```diff
@@ -449,25 +461,32 @@ class Linux(Platform):
     BASE_DYN_ADDR_32 = 0x56555000
     BASE_DYN_ADDR = 0x555555554000
 
-    def __init__(self, program, argv=None, envp=None, disasm="capstone", **kwargs):
+    def __init__(
+        self,
+        program: Optional[str],
+        argv: List[str] = [],
+        envp: List[str] = [],
+        disasm: str = "capstone",
+        **kwargs,
+    ):
         """
         Builds a Linux OS platform
         :param string program: The path to ELF binary
         :param string disasm: Disassembler to be used
         :param list argv: The argv array; not including binary.
         :param list envp: The ENV variables.
         :ivar files: List of active file descriptors
-        :type files: list[Socket] or list[File]
+        :type files: list[Socket or File or None]
         """
         super().__init__(path=program, **kwargs)
 
         self.program = program
         self.clocks = 0
-        self.files = []
+        self.files: List[Union[None, File, Socket]] = []
```

Comment:
Could this be
```suggestion
        self.files: List[Optional[Union[File, Socket]]] = []
```

Reply:
Yes, in general, `Optional[T]` is a synonym for `Union[None, T]` in mypy.